spec/01-unit/01-db/01-schema/11-declarative_config/02-process_auto_fields_spec.lua

```diff
@@ -23,6 +23,7 @@ describe("declarative config: process_auto_fields", function()
         config = DeclarativeConfig:process_auto_fields(config, "select", false)
         assert.same({
           _format_version = "1.1",
```

Comment:
What about these?

Reply:
Those are fine. The 1.2 can understand and accept 1.1-format. It is ok to test that if you put 1.1 into the process_auto_fields function, you get a 1.1 version back.